modes/longitudinal/src/index.js

```diff
@@ -108,9 +117,15 @@ export default function mode({ modeConfiguration }) {
       'org.ohif.measurement-tracking',
       'org.ohif.dicom-sr',
       'org.ohif.dicom-video',
+      'org.ohif.dicom-pdf',
     ],
     hangingProtocols: [ohif.hangingProtocols],
-    sopClassHandlers: [dicomvideo.sopClassHandler, ohif.sopClassHandler, dicomsr.sopClassHandler,],
+    sopClassHandlers: [
+      dicomvideo.sopClassHandler,
```

Comment:
can we sort this? maybe starting with `ohif.` one just for readability

Reply:
No, you can't sort it, the order is important because the first one that grabs an object takes it out of the available list.

Reply:
So if a mode say `[ohif.sopClassHandler, dicomvideo.sopClassHandler]` instead of `[dicomvideo.sopClassHandler, ohif.sopClassHandler]` it breaks? I don't understand, can you maybe point to the source of the problem?

Reply:
Yes, it breaks.  The problem is that ohif.sopClassHandler handles some of the same objects as dicomvideo.sopClassHandler, because it doesn't know about the available transfer syntax UID, so it will grab them first and add them to display sets.  Then, dicomvideo never gets called because there isn't anything for it to handle.

Reply:
As requested, changed to make each item declare the requested ordering, so that the mode doesn't need to know about the ordering of extensions inside itself.  In fact, the mode no longer has to have it's own sop class handler list, as there is a generic one it can inherit from.